package.json

```diff
@@ -1,52 +1,63 @@
 {
-  "name": "demeine",
   "version": "0.4.0",
+  "name": "demeine",
   "description": "DDDD - Distributed Domain Driven Design",
-  "jshint": "^2.5.10",
-  "main": "index.js",
+  "author": "Surikat AB",
+  "license": "MIT",
+  "main": "dist/index.js",
+  "typings": "dist/index.d.ts",
+  "files": [
+    "dist",
+    "src"
+  ],
+  "engines": {
+    "node": ">=10"
+  },
   "scripts": {
-    "build": "tsc -p .",
-    "prepublishOnly": "npm run build",
-    "test": "mocha test/**/*.ts --require ts-node/register --recursive",
-    "test.watch": "mocha --require ts-node/register --color --watch-extensions ts --watch test/**/*.ts",
-    "coverage": "istanbul cover node_modules/mocha/bin/_mocha -- --require ts-node/register --recursive test -u exports -R spec",
-    "coverage-start": "istanbul cover node_modules/mocha/bin/_mocha test --require ts-node/register --recursive -- -u exports -R spec && start coverage/lcov-report/index.html",
-    "lint": "tslint -p tsconfig.json",
-    "lint.fix": "tslint -p tsconfig.json --fix src/**/*.{ts,tsx} -t verbose"
+    "start": "tsdx watch",
+    "build": "tsdx build",
+    "test": "tsdx test",
+    "lint": "tsdx lint src",
+    "prepare": "tsdx build",
+    "test.watch": "tsdx test --watchAll",
+    "size": "size-limit",
+    "analyze": "size-limit --why"
   },
-  "repository": {
-    "type": "git",
-    "url": "https://github.com/surikaterna/demeine.git"
+  "peerDependencies": {},
+  "husky": {
+    "hooks": {
+      "pre-commit": "tsdx lint"
+    }
```

Comment:
This can be removed, right? We're not using husky.

Reply:
But we are here though. From now